### PR TITLE
Ps about storyboard

### PR DESCRIPTION
**Pull request by petestewart**

### CODE REVIEW

**Review by MLogan1998 — APPROVED** [review]
looks good to me

File: main.js [hunk 1]
```diff
@@ -167,12 +183,29 @@ const clickEvents = () => {
   }
 }
 
+const buildAboutFrames = () => {
+  console.log('running buildAboutFrames');
```
Comment (wcampbell91) [inline main.js]: delete this console log!

**Review by wcampbell91 — CHANGES_REQUESTED** [review]
remove console log and add new line at end of JS file

**Review by wcampbell91 — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (petestewart) [reply main.js]: Haha Done!

**Pull request merged**